source/common/signal/fatal_error_handler.cc

```diff
@@ -56,7 +55,7 @@ void runFatalActionsInternal(const FatalAction::FatalActionPtrList& actions) {
 
   // Restore the fatal_error_handlers pointer so subsequent calls using the list
   // can succeed.
-  fatal_error_handlers.store(list, std::memory_order_release);
+  fatal_error_handlers.store(list, std::memory_order_seq_cst);
```

Comment:
nit: IIRC `std::memory_order_seq_cst` is the default param for all atomic ops. Can you just remove it everywhere? It will make the code easier to read.

Reply:
Whoops. Done.